docs/cloud-native-application-developers-manual.md

```diff
@@ -0,0 +1,60 @@
+
+For BKPR to perform this automation application authors need to apply a few configurations in their application manifests. The following sections walk you through each of these configurations.
+
+## Ingress and Let's Encrypt
+
+BKPR installs a [nginx-ingress](HTTPS://github.com/kubernetes/ingress-nginx) controller to the cluster which provides load balancing, SSL termination and name-based virtual hosting for your applications. Ingress enables application developers to expose HTTP and HTTPS routes from outside the cluster to services running inside the cluster.
+
+BKPR performs DNS zone record updates for host entries present in [ingress rules](HTTPS://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules) automatically without any additional configurations.
+
+To automate letsencrypt certificate provisioning you should to add the annotation `kubernetes.io/tls-acme: "true"` to the ingress resource of your application. Optionally you could add the annotation `kubernetes.io/ingress.class: "nginx"` to designate that the ingress resource should only be handled by the NGINX controller.
+
+For example, with the following example snippet, BKPR will automatically update the DNS records for `myapp.mydomain.com` and request letsencrypt for a valid TLS certificate for your application. Following which you would be able to access the application securely over the internet.
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: "myapp-ingress"
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    kubernetes.io/tls-acme: "true"
+spec:
+  tls:
+  - hosts:
+    - myapp.mydomain.com
+    secretName: myapp-tls
+  rules:
+  - host: myapp.mydomain.com
+    http:
+      paths:
+        - path: /
+          backend:
+            serviceName: myapp-service
+            servicePort: 80
+```
+
+## Logging
+
+BKPR installs [Elasticsearch](https://elastic.co/products/elasticsearch) to the cluster which provides a RESTful search and analytics engine for processing your application logs. The stack uses [Fluentd](https://www.fluentd.org/) to capture the standard output of every container running in the cluster and persistent storage is used to store historical data of the cluster container logs.
+
+For Fluentd to be able to capture your applications logs, as a cloud native application developer, all you need to do is configure your application to output the logs to the standard output.
+
+The [Kibana](https://www.elastic.co/products/kibana) dashboard installed in the cluster can then be used to investigate and analyse abnormal behavior of your application running in the cluster.
+
+Please refer to the [NGINX Ingress Controller](components.md#nginx-ingress-controller) component documentation to learn more.
```

Comment:
Isn't this in the wrong place? It feels like it should belong in the previous section.